.all-contributorsrc

```diff
@@ -40,11 +40,19 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "zaidakhterr",
+      "name": "Zaid Akhter",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/51528814?v=4",
+      "profile": "https://github.com/zaidakhterr",
```

Comment:
Hi this was my first pull request. I probably could have done a bettter job. But if possible please add my linkden instead. Link below!
https://www.linkedin.com/in/zaidakhterr/